### PR TITLE
🏷️ Update Types  react-native-datawedge-intents

### PR DESCRIPTION
**Pull request by Frabanz**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: <https://github.com/darryncampbell/react-native-datawedge-intents/blob/master/index.js>
- [X] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/react-native-datawedge-intents/react-native-datawedge-intents-tests.ts [hunk 1]
```diff
@@ -24,3 +24,15 @@ function sendCommand(extraName: string, extraValue: any) {
...
+sendIntent('ACTION_SOFTSCANTRIGGER', 'START_SCANNING');
+
+function sendIntent(action: string, parameterValue: string) {
+    DataWedgeIntents.sendIntent(action, parameterValue);
+}
+
+registerReceiver('com.zebra.dwintents.ACTION', '');
+
+function registerReceiver(action: string, category: string) {
+DataWedgeIntents.registerReceiver(action, category);
```
Comment (Frabanz) [inline types/react-native-datawedge-intents/react-native-datawedge-intents-tests.ts:37]:
```suggestion
    DataWedgeIntents.registerReceiver(action, category);
```

File: types/react-native-datawedge-intents/index.d.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
-// Type definitions for react-native-datawedge-intents 0.1
+// Type definitions for react-native-datawedge-intents 0.1.6
```
Comment (Frabanz) [inline types/react-native-datawedge-intents/index.d.ts:1]:
```suggestion
// Type definitions for react-native-datawedge-intents 0.1
```

**Comment by orta** [comment]
Are you sure you want to include these? They are both marked as deprecated in the JS, and may have been left out on purpose.

**Comment by elibarzilay** [comment]
Ping re @orta's question

**Review by sandersn — CHANGES_REQUESTED** [review]
Requesting changes so the bot catches to the current state of the PR.

**Pull request closed without merging**